backend/.env.example

```
@@ -0,0 +1,35 @@
+FINNHUB_API_KEY=
+
+# --- Order Book / Market Depth ---
+# Crypto pairs (e.g. BTC-USD) return genuine Level-2 depth from Coinbase's
+# free public API — no key required, works out of the box.
+#
+# Stocks (e.g. NVDA) use Alpaca's free IEX top-of-book (Level 1). This needs a
+# free API key (the plain Key ID + Secret from the dashboard — NOT OAuth).
+# Create a free account at https://alpaca.markets, then under Paper Trading →
+# "API Keys" generate a key. When unset, GET /orderbook/{stock} returns 503,
+# but crypto pairs still work.
+ALPACA_API_KEY=        # Key ID from app.alpaca.markets (paper trading)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)